src/coq_rules.ml

```diff
@@ -0,0 +1,111 @@
+(* This file is licensed under The MIT License *)
+(* (c) MINES ParisTech 2018-2019               *)
+(*     Written by: Emilio Jesús Gallego Arias  *)
+
+open! Stdune
+open Build.O
+module SC = Super_context
+
+let coq_debug = false
+
+(* TODO:                                       *)
```

Comment:
I'd suggest doing this, opening an issue, or removing this comment :slightly_smiling_face:

Reply:
Comment removed; I will indeed open an issue (in the Coq repos) to discuss what library model should Dune + Coq support. There are a few design decisions to make.